parquet/src/main/java/org/apache/iceberg/parquet/ParquetValueReaders.java

```diff
@@ -63,6 +69,18 @@ public static ParquetValueReader<Long> position() {
     return new PositionReader();
   }
 
+  public static ParquetValueReader<UUID> uuids(ColumnDescriptor desc) {
+    return new ParquetValueReaders.UUIDReader(desc);
+  }
+
+  public static ParquetValueReader<Long> timestampInt96Reader(ColumnDescriptor desc) {
+    return new ParquetValueReaders.TimestampInt96Reader(desc);
+  }
+
+  public static ParquetValueReader<Long> timestampMillisReader(ColumnDescriptor desc) {
```

Comment:
I'd probably rename this as well, maybe `milliTimestamps` or `millisecondTimestamps`